diff -uNr cvsweb.orig/cvsweb.cgi cvsweb/cvsweb.cgi
--- cvsweb.orig/cvsweb.cgi	Sun Oct  3 19:24:58 1999
+++ cvsweb/cvsweb.cgi	Tue Oct 12 21:27:48 1999
@@ -73,7 +73,7 @@
 ##### Start of Configuration Area ########
 # == EDIT this == 
 # User configuration is stored in
-$config = $ENV{'CVSWEB_CONFIG'} || '/usr/local/web/apache/conf/cvsweb.conf';
+$config = $ENV{'CVSWEB_CONFIG'} || '/etc/cvsweb.conf';
 
 # == Configuration defaults ==
 # Defaults for configuration variables that shouldn't need
--- ./cvsweb.conf~	2005-04-02 03:00:23.000000000 +0300
+++ ./cvsweb.conf	2005-04-02 03:01:29.000000000 +0300
@@ -23,14 +23,15 @@
 
 # 'symbolic_name' 'path_to_the_actual_repository'
 %CVSROOT = (
-	    'Development' => '/usr/local/src/cvsrep',
-	    'Configuration' => '/tmp/cvsroot/conf',
-	    'HTML-files' => '/tmp/upload'
+# Uncomment next line and modify the path if you have only one CVS repository.
+#	    'Development' => '/usr/local/src/cvsrep',
+#	    'Configuration' => '/tmp/cvsroot/conf',
+#	    'HTML-files' => '/tmp/upload'
 	    );
 
 # This tree is enabled by default when
 # you enter the page
-$cvstreedefault = 'Configuration';
+$cvstreedefault = 'Local-CVS';
 
 ##############
 # Defaults for UserSettings
@@ -81,7 +82,7 @@
 ##############
 
 # color settings in the body-tag
-$body_tag = '<body text="#000000" bgcolor="#ffffff">';
+$body_tag = '<body text="#000000" bgcolor="#ffffff" link="#0000ff" alink="#ff0000" vlink="#800080">';
 
 # Wanna have a logo on the page ?
 $logo = '<img src="/icons/apache_pb.gif">';
@@ -291,6 +292,8 @@
 	   "gif"   => "image/gif",
 	   "jpeg"  => "image/jpeg",
 	   "jpg"   => "image/jpeg",   
+	   "png"   => "image/png",
+	   "xpm"   => "image/xpm",
 	   "*"	   => "text/plain",
 	   );
 
--- ./cvsweb.conf~	2005-04-02 03:05:40.000000000 +0300
+++ ./cvsweb.conf	2005-04-02 03:10:52.000000000 +0300
@@ -279,7 +279,7 @@
 # cvsweb to guess the correct mime-type on
 # checkout; you can use the mime.types from
 # apache here:
-$mime_types = '/usr/local/web/apache/conf/mime.types';
+$mime_types = '/etc/mime.types';
 
 # quick mime-type lookup; maps file-suffices to
 # mime-types for displaying checkouts in the browser.
