--- a/cvsweb.cgi
+++ b/cvsweb.cgi
@@ -214,7 +214,8 @@
 
 # == EDIT this ==
 # Locations to search for user configuration, in order:
-for (catfile($mydir, 'cvsweb.conf'), '/usr/local/etc/cvsweb/cvsweb.conf') {
+my ($CVSWEB_CONFIG) = $1 if $ENV{CVSWEB_CONFIG} =~ /(.*)/; # untaint
+for ($CVSWEB_CONFIG, catfile($mydir, 'cvsweb.conf'), '/etc/webapps/cvsweb/cvsweb.conf') {
   if (-r $_) {
     $config = $_;
     last;
--- a/cvsweb.conf
+++ b/cvsweb.conf
@@ -24,7 +24,7 @@
 #   cvsgraph (if you enable $allow_graph)
 #   enscript (if you enable $allow_enscript)
 #
-@command_path = qw(/bin /usr/bin /usr/local/bin);
+@command_path = qw(/bin /usr/bin);
 
 # Search the above directories for each command (prefer gtar over tar).
 #
@@ -57,7 +57,7 @@
 # 'symbolic_name' => ['Name to display',  '/path/to/cvsroot']
 #
 @CVSrepositories = (
-        'local'   => ['Local Repository', '/var/cvs'],
+        'local'   => ['Local Repository', '/home/services/cvsroot'],
 #       'freebsd' => ['FreeBSD',          '/var/ncvs'],
 #       'openbsd' => ['OpenBSD',          '/var/ncvs'],
 #       'netbsd'  => ['NetBSD',           '/var/ncvs'],
@@ -72,7 +72,7 @@
 # For example:
 #
 #$cvstreedefault = $CVSrepositories[2 * 0];
-#$cvstreedefault = 'local';
+$cvstreedefault = 'local';
 
 # Mirror sites.  The keys will be used as link texts, and the values are
 # URLs pointing to the corresponding mirrors.
@@ -106,7 +106,7 @@
 # $allow_*_extra below is true.
 #
 $mancgi =
-    "http://www.FreeBSD.org/cgi/man.cgi?apropos=0&sektion=%s&query=%s&manpath=FreeBSD+5.0-current&format=html";
+    "http://linux.die.net/man/%s/%s";
 
 # Defaults for user definable options.
 #
@@ -226,7 +226,7 @@
 # check out the "mini" versions in the icons/ directory; they have a
 # width/height of 16/16.
 #
-my $iconsdir = '/icons';
+my $iconsdir = '/cvsweb/icons';
 
 # format:          TEXT       ICON-URL                  width height
 %ICONS = (
@@ -240,7 +240,7 @@
 
 # An URL where to find the CSS.
 #
-$cssurl = '/css/cvsweb.css';
+$cssurl = '/cvsweb/css/cvsweb.css';
 
 # The length to which the last log entry should be truncated when shown
 # in the directory view.
@@ -368,7 +368,7 @@
 # The traditional mime.types file, eg. the one from Apache is fine.
 # See above where this gets used.
 #
-$mime_types = '/usr/local/etc/apache/mime.types';
+$mime_types = '/etc/mime.types';
 
 # Charset appended to the Content-Type HTTP header for text/* MIME types.
 # Note that the web server may default to some charset which may take effect
@@ -505,7 +505,7 @@
 # settings in the configuration file with command line options, see
 # doGraph() and doGraphView() in cvsweb.cgi for details.
 #
-#$cvsgraph_config = "/etc/cvsgraph.conf";
+#$cvsgraph_config = "/etc/cvsgraph/cvsgraph.conf";
 
 # URL to the CVSHistory script.  This should be absolute (but does not need
 # to include the host and port if the script is on the same server as
@@ -610,6 +610,7 @@
    'nroff'        => qr/\.man$/o,
    'pascal'       => qr/\.p(as|p)?$/io,
    'perl'         => qr/\.p(m|(er)?l)$/io,
+   'php'          => qr/\.php$/io,
    'postscript'   => qr/\.e?ps$/io,
    'python'       => qr/\.py$/o,
    'rfc'          => qr/\b((rfc|draft)\..*\.txt)$/o,
