--- ./cvsweb.cgi~	2005-04-02 03:03:37.000000000 +0300
+++ ./cvsweb.cgi	2005-04-02 03:03:57.000000000 +0300
@@ -164,6 +164,7 @@
 $verbose = $v;
 $checkoutMagic = "~checkout~";
 $where = defined($ENV{'PATH_INFO'}) ? $ENV{'PATH_INFO'} : "";
+$where =~ tr|/|/|s;
 $doCheckout = ($where =~ /^\/$checkoutMagic/);
 $where =~ s|^/($checkoutMagic)?||;
 $where =~ s|/+$||;
@@ -936,6 +937,7 @@
 sub link($$) {
 	my($name, $where) = @_;
 
+	$where =~ s| |%20|;
 	return "<A HREF=\"$where\">$name</A>\n";
 }
 
--- ./cvsweb.cgi~	2005-04-02 03:11:28.000000000 +0300
+++ ./cvsweb.cgi	2005-04-02 03:18:05.000000000 +0300
@@ -1253,6 +1253,10 @@
 	}
     }
     
+	# remove first slash
+	# otherwise we get "Absolute module reference invalid" error from cvs
+    $where =~ s#^/##;
+
     ### just for the record:
     ### 'cvs co' seems to have a bug regarding single checkout of
     ### directories/files having spaces in it;
