diff -uNr cvsweb.orig/cvsweb.cgi cvsweb/cvsweb.cgi
--- cvsweb.orig/cvsweb.cgi	Sun Oct  3 19:24:58 1999
+++ cvsweb/cvsweb.cgi	Tue Oct 12 21:27:48 1999
@@ -73,7 +73,7 @@
 ##### Start of Configuration Area ########
 # == EDIT this == 
 # User configuration is stored in
-$config = $ENV{'CVSWEB_CONFIG'} || '/usr/local/web/apache/conf/cvsweb.conf';
+$config = $ENV{'CVSWEB_CONFIG'} || '/etc/cvsweb.conf';
 
 # == Configuration defaults ==
 # Defaults for configuration variables that shouldn't need
